src/components/Card.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { imgUrl } from "../urlStrings";
+
+const Card = ({ id, title, poster_path, buttonDisable, addMovieButton }) => {
+    
+    return (
+        <div>
+            <div className="card" id="movie-cards">
+                <button id="add-movie-button" className="btn btn-secondary btn-sm" onClick={addMovieButton} value={id} disabled={buttonDisable}>Add</button>
```

Comment:
We also need the add button to be transformed to a remove button. When the user adds a movie into the cart, the add button on the movie card should be a remove button and when the user clicks on it, the movie should be removed from the cart.